src/conditionalAndLoops.js

```diff
@@ -168,10 +236,22 @@ function timespanToHumanString(startDate, endDate) {
  *    365, 4  => '11231'
  *    365, 10 => '365'
  */
+let str = "";
+function generate(num, n) {
+	if (num < n) {
+		str += num;
+		return;
+	}
+	generate(Math.floor(num / n), n);
+	str += (num % n);
+}
 function toNaryString(num, n) {
-	throw new Error("Not implemented");
+	str = "";
+	generate(num, n, str);
+	return str;
```

Comment:
LGTM. Try to update the implementation to support bases till 16.

Reply:
Okay bhaiya. Thank you for pointing out.

let str = "";
let converter = {
	10: "a",
	11: "b",
	12: "c",
	13: "d",
	14: "e",
	15: "f"
}
function generate(num, n) {
	if (num < n) {
		str += num;
		return;
	}
	generate(Math.floor(num / n), n);
	if(num%n < 10)
	{
		str += (num % n);
	}
	else
	{
		str += (converter[num%n]);
	}
}
function toNaryString(num, n) {
	str = "";
	generate(num, n, str);
	return str;
}

Is this correct bhaiya?